File: C++/C++Basic/03Conditional-Statements-Lab/07.areaOfFigures.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){
    string figure;
    cin >> figure;

    double area = 0;

    if (figure == "square"){
        double side;
        cin >> side;
        area = side * side;
    }
    else if (figure == "rectangle"){
        double sideA, sideB;
        cin >> sideA;
        cin >> sideB;

        area = sideA * sideB;

    }
    else if (figure == "circle"){
        double radius;
        cin >> radius;

        area = 3.14159 * radius * radius;
    }
    else if (figure == "triangle")
    {
        double length, h;
        cin >> length;
        cin >> h;

        area = h/2 * length;
    }
    

    cout.setf(ios::fixed);
    cout.precision(3);
    
    cout << area << endl;
    
    return 0;
}

File: Python/Basic/03.Conditional-Statements-Lab/07.area_of_figures.py
from math import pi, pow

figure = input()
area = 0

if figure == "square":
    sideA = float(input())
    area = sideA * sideA
elif figure == "rectangle":
    sideA = float(input())
    sideB = float(input())
    area = sideA * sideB
elif figure == "circle":
    radius = float(input())
    area =  pi * pow(radius, 2)
elif figure == "triangle":
    length = float(input())
    height = float(input())
    area =(height * length)/ 2
print(f'{area:.3f}')
